### PR TITLE
feat: support for profile v2

### PR DESCRIPTION
**Pull request by idoshamun**
* User's readme support including rendering the readme to markdown
* Cover image support including uploading to cloudinary
* Expose stats to everyone

### CODE REVIEW

File: src/routes/boot.ts [hunk 1]
```diff
@@ -334,6 +334,31 @@ const getExperimentation = async (
...
+      'id',
+      'username',
+      'name',
+      'email',
+      'image',
+      'company',
+      'title',
+      'infoConfirmed',
+      'notificationEmail',
+      'acceptedMarketing',
```
Comment (rebelchris) [inline src/routes/boot.ts:350]: Do we need these in the fe from boot?

File: src/schema/users.ts [hunk 2]
```diff
@@ -661,7 +709,6 @@ export const resolvers: IResolvers<any, Context> = {
       { id }: { id: string },
       ctx: Context,
     ): Promise<GQLUserStats | null> => {
-      const isSameUser = ctx.userId === id;
```
Comment (rebelchris) [inline src/schema/users.ts:664]: We no longer want this eval?

**Review by rebelchris — COMMENTED** [review]
Small questions

File: src/schema/users.ts (hunk 2 above)
Comment (idoshamun) [reply src/schema/users.ts:664]: Nope, we gonna show these metrics publicly for now

File: src/routes/boot.ts (hunk 1 above)
Comment (idoshamun) [reply src/routes/boot.ts:350]: So, I didn't want to take risks in this PR, but I definitely know we don't need the new fields I added, which can be quite big. Hence, I manually selected what we had before. I think we should optimize it and see what's needed, but that's out of this scope. The previous implementation just selected everything

**Pull request merged**